src/main/java/com/app/onlinebookstore/model/Book.java

```diff
@@ -0,0 +1,97 @@
+package com.app.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+
+@SuppressWarnings("unused")
+@Entity
+@Table(name = "book")
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private String title;
+    @Column(nullable = false)
+    private String author;
+    @Column(nullable = false, unique = true)
+    private String isbn;
```

Comment:
isbn? what is that?

Reply:
According to the requirements:

![image](https://github.com/Dimagaa/online-book-store/assets/113283324/a171751a-0d9b-49b2-9e41-7df66570830b)